src/GenesisPreferencesWindow.cpp

```diff
@@ -35,6 +38,11 @@ GenesisPreferencesWindow::GenesisPreferencesWindow(BLooper* looper, BWindow *mai
 	m_SymlinkedPaths = new BCheckBox("followsymlinks", "Keep symlinked directory paths when navigating", new BMessage(PREFERENCES_CHANGED));
 	m_AskOnExit = new BCheckBox("askonexit", "Ask on exit", new BMessage(PREFERENCES_CHANGED));
 
+	m_EditorsMenu = new BMenu("choose ...",B_ITEMS_IN_COLUMN);
```

Comment:
```suggestion
	m_EditorsMenu = new BMenu("Select ...",B_ITEMS_IN_COLUMN);
```

You can also replace "..." with B_UTF8_ELLIPSIS everywhere.